src/app/api/core/services/policies.service.ts

```diff
@@ -10,11 +10,11 @@ import httpStatus from 'http-status'
 export class PoliciesService extends BaseService {
   private defaultPolicies: Record<UserRole.Client, Record<Resource, UserAction[]>> = {
     [UserRole.Client]: {
-      [Resource.Tasks]: [UserAction.Read],
+      [Resource.Tasks]: [UserAction.Read, UserAction.Update],
       [Resource.TaskTemplates]: [],
       [Resource.WorkflowState]: [UserAction.Read],
       [Resource.ViewSetting]: [],
-      [Resource.Users]: [],
+      [Resource.Users]: [UserAction.Read],
```

Comment:
Same here. This would allow users to view data of all of the users associated with a workspace, including user data from other organizations